packages/paste-icons/tools/utils.js

```diff
@@ -1,10 +1,13 @@
 const {join} = require('path');
 const fs = require('fs');
-const startcase = require('lodash.startcase');
+const startcase = require('lodash/startcase');
```

Comment:
one thing I couldn't figure out: we use lodash here, but it's not in the package.json for the paste-icons package. When I tried to add it, I got a syntax error at the top of the package.json file:
<img width="654" alt="image" src="https://user-images.githubusercontent.com/77117846/169560772-4544400c-0197-4e64-8e09-8e11c754c649.png">

I removed it to be able to push up and have the other changes reviewed. But any ideas on why that error would've come up for adding lodash as a dependency in the paste-icons package.json?

Reply:
I'm adding it under a new "dependency" key, since that's the way it is in the other packages: 
<img width="273" alt="image" src="https://user-images.githubusercontent.com/77117846/169890173-5e48e728-5615-48a0-8fe7-0284573b7077.png">